pmm-app/src/pmm-qan/panel/components/Details/Details.tsx

```diff
@@ -61,57 +60,65 @@ export const DetailsSection: FC = () => {
+      component: <Explain examples={examples} databaseType={databaseType} />,
+    },
+    {
+      label: Messages.tabs.tables.tab,
+      key: TabKeys.tables,
+      show: showTablesTab,
+      component: <TableCreateContainer databaseType={databaseType} examples={examples} database={database} />,
+    },
+  ];
+
   return (
     <Scrollbar className={styles.scrollArea}>
       <div className={cx(styles.detailsGrid, 'query-analytics-details')} data-qa="query-analytics-details">
         <div className="details-tabs">
-          <Divider className={styles.zeroMargin} />
-          <Tabs
-            activeKey={activeTab}
-            onChange={(newTab) => {
-              changeActiveTab(newTab);
-              setActiveTab(newTab);
-            }}
-            tabPosition="top"
-            destroyInactiveTabPane
-            tabBarExtraContent={(
-              <Button type="default" size="small" onClick={closeDetails}>
-                {Messages.closeDetails}
-              </Button>
-            )}
-          >
-            <TabPane tab={<span>{Messages.tabs.details.tab}</span>} key={TabKeys.details}>
-              <Metrics
-                databaseType={databaseType}
-                totals={totals}
-                metrics={metrics}
-                loading={metricsLoading}
-              />
-            </TabPane>
-            {showExamplesTab ? (
-              <TabPane tab={<span>{Messages.tabs.examples.tab}</span>} key={TabKeys.examples}>
-                <Example
-                  databaseType={databaseType}
-                  examples={examples}
-                  loading={loading || metricsLoading}
+          <TabsBar>
+            {tabs
+              .filter(({ show }) => show)
+              .map((tab, index) => (
+                <Tab
+                  key={index}
+                  label={tab.label}
+                  active={tab.key === activeTab}
+                  onChangeTab={() => {
+                    changeActiveTab(tab.key);
+                    setActiveTab(tab.key);
+                  }}
                 />
-              </TabPane>
-            ) : null}
-            {showExplainTab ? (
-              <TabPane
-                tab={<span>{Messages.tabs.explains.tab}</span>}
-                key={TabKeys.explain}
-                disabled={totals}
-              >
-                <Explain examples={examples} databaseType={databaseType} />
-              </TabPane>
-            ) : null}
-            {showTablesTab ? (
-              <TabPane tab={<span>{Messages.tabs.tables.tab}</span>} key={TabKeys.tables} disabled={totals}>
-                <TableCreateContainer databaseType={databaseType} examples={examples} />
-              </TabPane>
-            ) : null}
-          </Tabs>
+              ))}
+            <Button
+              variant="secondary"
+              size="sm"
+              onClick={closeDetails}
+              style={{ marginLeft: 'auto', marginRight: '10px', alignSelf: 'center' }}
```

Comment:
Consider putting this in styles file and using className.

Reply:
Done